app/src/main/res/layout/view_places_filter_item.xml

```diff
@@ -3,42 +3,65 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="horizontal"
+    xmlns:tools="http://schemas.android.com/tools"
     android:background="?attr/paper_color"
-    android:paddingTop="16dp">
+    android:orientation="vertical">
 
-    <TextView
-        android:id="@+id/placesFilterLangCode"
-        style="@style/TextViewCentered"
-        android:layout_width="24dp"
-        android:layout_height="24dp"
-        android:layout_gravity="center_vertical"
-        android:layout_marginStart="24dp"
-        android:layout_marginEnd="-10dp"
-        android:background="@drawable/lang_button_shape_border"
-        android:backgroundTint="?attr/primary_color"
-        android:fontFamily="sans-serif-medium"
-        android:textAllCaps="true"
-        android:textColor="?attr/primary_color" />
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="48dp"
+        android:background="?selectableItemBackground"
+        android:orientation="horizontal">
 
-    <TextView
-        android:id="@+id/placesFilterTitle"
-        style="@style/H3"
-        android:layout_width="0dp"
-        android:layout_height="46dp"
-        android:layout_weight="1"
-        android:layout_marginStart="24dp"
-        android:gravity="center_vertical"
-        android:textColor="?attr/primary_color" />
+        <LinearLayout
+            android:id="@+id/placesFilterLangContainer"
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:paddingHorizontal="10dp"
+            android:layout_marginStart="14dp"
+            android:background="?attr/selectableItemBackground"
```

Comment:
This view doesn't need its own `selectableItemBackground`, since the parent view already has it.